ert3/console/_console.py

```diff
@@ -36,6 +36,13 @@ def _build_init_argparser(subparsers: Any) -> None:
 def _build_run_argparser(subparsers: Any) -> None:
     run_parser = subparsers.add_parser("run", help="Run experiment")
     run_parser.add_argument("experiment_name", help="Name of the experiment")
+    run_parser.add_argument(
+        "--local-test-run",
+        default=False,
+        action="store_true",
+        help="Will evaluate a single realization locally in the "
+        "folder `<workspace root>/local-test-run`.",
```

Comment:
I think we should mention the numbers that will be added to this path as well here